src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,220 @@
+trigger:
+- master
+- main
+- release
+
+# trigger this pipeline if there's a PR to any of these branches
+pr:
+- master
+- main
+- staging
+- releases/*
+
+variables:
+- name: jobTimeout
+  value: '60'
+
+stages:
+  - stage: Build_UNO
```

Comment:
Currently the Build_UNO stage has two steps for android/ios/windows, with the second step being the creation of the signed package. I was wondering whether we should push the second step into a separate stage that would only be done on successful completion of the first stage. This way we're not generating signed packages unless all the platforms build successfully.
cc @dansiegel @jeromelaban  thoughts?

Reply:
Signing is generally best to be done on a separate stage to avoid rebuilding multiple times on signing failures (in case a provisioning profile needs to be provided again)

Reply:
It is one thing if we want to produce an unsigned artifact for a PR but when this runs for a merge then it should always produce a signed artifact otherwise what's the point of the template. The notion that you need to wait and see if all of the platforms build successfully and then go back and build it a second time to produce a signed version would be a waste of agent build minutes.

The more typical for a merge is build everything, sign what needs to be signed... then if they all build successfully you trigger a deployment stage which could be to say QA... then if that is approved you might rebuild your app using a different environment so that your app hits your prod API, etc... then deploy to prod... In any case the default should always be produce signed builds with the exception of maybe for a PR....